src/app/Models/Product.php

```diff
@@ -47,4 +44,14 @@ public function defaultSupplier()
             ->wherePivot('is_default', true)
             ->first();
     }
+
+    public function syncSuppliers($supplierIds, $defaultSupplierId)
```

Comment:
find where we do the same in other packages and upgrade with this implementations